### ISSUE TITLE
Fix `aria-required-children` violations 

### ISSUE THREAD

**Issue by DanielCliftonGuardian** [guardian/dotcom-rendering#15068]
WCAG: 2.0 A (4.1.2)
Affected: 2 stories
Fix: Ensure ARIA parent roles have required child roles

Affected Components
Components/Carousel (2 stories)
